class SkillExistsError(Exception):
    pass


raise SkillExistsError("Skill already exists")
